woocommerce/woocommerce
# Add WooCommerce Payments banner to extensions screen

### All Submissions:






### Changes proposed in this Pull Request:
Introduce a similar banner as the WC Shipping one for WC Pay on the extensions screen under featured and the payments sections, geo target it to US stores only.

- New WC Pay banner on featured extensions section
- New WC Pay banner on the payments extension section
- Update button colors to new purple
- Install WC Pay from wp.org when clicking the banner install button
- Also got a request to update the services naming to WooCommerce Shipping.

paqDfi-Ru-p2

### How to test the changes in this Pull Request:

1. Checkout branch locally and build assets
2. Log into your wp-admin and ensure your store location is set to US.
3. Head to `WooCommerce -> Extensions`
4. Check that the WooCommerce Pay banner is showing
5. Click the Payments subheading and check the WooCommerce Pay banner is showing.
6. Test the install functionality of the button on the banner
7. Ensure the banner is hidden when WooCommerce Pay is installed and activated.

If your store location is set to the US and the banner is not showing on the featured section you will have to clear the `wc_addons_featured` transient as the instruction to show the banner is handled via the featured json file.

![](https://d.pr/i/ezx8nR+)
![](https://d.pr/i/9L3e0T+)

### Other information:




### Changelog entry

> New - Feature WooCommerce Payments in the extensions store for US-based stores.
